kaznet/apps/main/tests/viewsets/test_submissions.py

```diff
@@ -670,7 +671,8 @@ def test_submission_time_sorting(self):
             response.data['results'][-1]['submission_time'],
             Submission.objects.order_by(
```

Comment:
I don't understand why we need to manually remove microseconds from the Queryset.

Reply:
Because It's comparing data coming from the `Viewset` with data coming from the Submission model which are now different with regards to microseconds.

Reply:
These assert statements still expected that data to be the same, so I though, now that the data is not the same, the tests should be updated too.